### ISSUE TITLE
resnet_50 CTL-TFv1.14(possible error?)

### ISSUE THREAD

**Issue by capilano** [tensorflow/tpu#479]
The code scales the loss(within the strategy scope) and then updates the loss using tf.keras.metrics.mean, I think tf.keras,metrics.Sum should be used to  update instead. I tried running  a CycleGAN like model using this example as a starting point and found that the loss values were very low compared to what I was getting using estimators but my generated images look similar to what I had previously.
